includes/html/pages/device/overview.inc.php

```diff
@@ -40,36 +41,9 @@
 require 'overview/mempools.inc.php';
 require 'overview/storage.inc.php';
 require 'overview/toner.inc.php';
-require 'overview/sensors/charge.inc.php';
-require 'overview/sensors/temperature.inc.php';
-require 'overview/sensors/humidity.inc.php';
-require 'overview/sensors/fanspeed.inc.php';
-require 'overview/sensors/dbm.inc.php';
-require 'overview/sensors/voltage.inc.php';
-require 'overview/sensors/current.inc.php';
-require 'overview/sensors/runtime.inc.php';
-require 'overview/sensors/power.inc.php';
-require 'overview/sensors/power_consumed.inc.php';
-require 'overview/sensors/power_factor.inc.php';
-require 'overview/sensors/frequency.inc.php';
-require 'overview/sensors/load.inc.php';
-require 'overview/sensors/state.inc.php';
-require 'overview/sensors/count.inc.php';
-require 'overview/sensors/percent.inc.php';
-require 'overview/sensors/signal.inc.php';
-require 'overview/sensors/tv_signal.inc.php';
-require 'overview/sensors/bitrate.inc.php';
-require 'overview/sensors/airflow.inc.php';
-require 'overview/sensors/snr.inc.php';
-require 'overview/sensors/pressure.inc.php';
-require 'overview/sensors/cooling.inc.php';
-require 'overview/sensors/delay.inc.php';
-require 'overview/sensors/quality_factor.inc.php';
-require 'overview/sensors/chromatic_dispersion.inc.php';
-require 'overview/sensors/ber.inc.php';
-require 'overview/sensors/eer.inc.php';
-require 'overview/sensors/waterflow.inc.php';
-require 'overview/sensors/loss.inc.php';
+foreach (Sensor::getTypes() as $sensor_class) {
+    require 'overview/sensors.inc.php';
```

Comment:
Requires same file. Also need a check if the file exists

Reply:
Thanks!

Reply:
I need to require multiple times. Because of historic reasons I need to give the $sensor_class in the inc.php file. I am not that skilled yet to migrate this to laravel blade. So step by step...

But without the File the GUI should break because it's essential, so I see no need to check this or are there other reasons I don't know?